src/lib/style/components/_query-wrapper-plot-nav.scss

```diff
@@ -2,9 +2,11 @@
 @use 'sass:map';
 
 .QueryWrapperPlotNav {
+  .SRC-wrapper {
+    display: grid;
```

Comment:
Used to only have a grid layout when in desktop mode (so in mobile the layout was based on the html output, which was not quite right).

Reply:
Namely, I saw the search bar above the top level controls.  There may have been other issues